lib/data/knowledge.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "title": "How do I whitelist my Vercel deployment IP address?",
```

```suggestion
    "title": "How do I identify my deployment IP address?",
```

Maybe this title could be changed?